cmd/swarmctl/service/flagparser/restart.go

```diff
@@ -72,6 +76,10 @@ func parseRestart(flags *pflag.FlagSet, spec *api.ServiceSpec) error {
 			return err
 		}
 
+		if windowDelay < time.Duration(0) {
```

Comment:
This should be checked here:

https://github.com/docker/swarmkit/blob/master/manager/controlapi/service.go#L45